cursor_pagination.py

```diff
@@ -36,6 +36,9 @@ def __getitem__(self, key):
     def __repr__(self):
         return '<Page: [%s%s]>' % (', '.join(repr(i) for i in self.items[:21]), ' (remaining truncated)' if len(self.items) > 21 else '')
 
+    async def acount(self):
+        return await self.items.acount()
```

Comment:
At the point where the CursorPage is created, `items` should be a list so I don't think this would work as intended

Reply:
uhm, interesting, the tests are passing though, I'll double check 😊

Reply:
![CleanShot 2024-06-17 at 20 44 46@2x](https://github.com/photocrowd/django-cursor-pagination/assets/667029/cb747752-6e8b-4090-9bcb-c9e9e4fbb13e)

Reply:
@bradleyoesch do you where we might be converting items to a list?

Reply:
Huh.

* In `page()`, we create `qs = list(qs)` then slice it `items = qs[:page_size]`, and pass that list into `CursorPage` as `items`
* In `apage()`, we create `items = []` and append to it, and pass that list into `CursorPage` as `items`

I'm having trouble recreating the above screenshot locally. What did you run?

Reply:
Ahhh

https://github.com/photocrowd/django-cursor-pagination/blob/ccc959fb8db6d3a084fe41dbfdc743f7a5f95a3e/cursor_pagination.py#L80

```python
        if page_size is None:
            return CursorPage(qs, self)
```

this could probably just be 

```python
        if page_size is None:
            return CursorPage([], self)
```

Reply:
Actually it should return the unfiltered queryset if there is no first or last
```python
        if page_size is None:
            return qs
```

But also if first or last is 0 it should still process it to truncate the queryset to 0, which I think right now it will skip since `0 or None` evaluates to `None`

Reply:
feel free to send a commit for that, just saw this and it's a bit late  for me :D 

btw I did get that screenshot by running the tests locally 😊

Reply:
Ok got the tests running so I can confirm this works! Sorry for all the back and forth

Ended up removing the `if page_size is None` block since it effectively skips filtering anyway, changed `first or last` to be more explicit about `None`, and added 0 first/last tests